src/puppetlabs/puppetdb/query/population.clj

```diff
@@ -12,7 +12,7 @@
   {:post [(number? %)]}
   (-> (str "SELECT COUNT(*) AS c "
            "FROM catalogs clogs, catalog_resources cr, certnames c "
-           "WHERE clogs.id=cr.catalog_id AND c.certname=clogs.certname AND c.deactivated IS NULL")
+           "WHERE clogs.id=cr.catalog_id AND c.certname=clogs.certname AND c.deactivated IS NULL AND c.expired IS NULL")
```

Comment:
but that doesn't indent *nearly* as well.

On Fri, May 8, 2015 at 1:34 PM, Wyatt Alt <notifications@github.com> wrote:

> In src/puppetlabs/puppetdb/query/population.clj
> <https://github.com/puppetlabs/puppetdb/pull/1377#discussion_r29974253>:
>
> > @@ -12,7 +12,7 @@
> >    {:post [(number? %)]}
> >    (-> (str "SELECT COUNT(*) AS c "
> >             "FROM catalogs clogs, catalog_resources cr, certnames c "
> > -           "WHERE clogs.id=cr.catalog_id AND c.certname=clogs.certname AND c.deactivated IS NULL")
> > +           "WHERE clogs.id=cr.catalog_id AND c.certname=clogs.certname AND c.deactivated IS NULL AND c.expired IS NULL")
>
> I think you should be able to just break this string over multiple lines
> (without str or multiple quotes)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetdb/pull/1377/files#r29974253>.
>